index.js

```diff
@@ -26,11 +26,43 @@ class Client {
         commands: {
           deploy: {
             usage: 'Deploy serverless client code',
-            lifecycleEvents: ['deploy']
+            lifecycleEvents: ['deploy'],
+            options: {
+              region: {
+                usage: 'Specify what AWS region the bucket will be deployed in',
+                type: 'string'
+              },
+              'delete-contents': {
+                usage: 'Delete the bucket contents before deployment',
+                type: 'boolean'
+              },
+              'config-change': {
+                usage: 'Overwrite the bucket configuration',
+                type: 'boolean'
+              },
+              'policy-change': {
+                usage: 'Overwrite the bucket policy',
+                type: 'boolean'
+              },
+              'cors-change': {
+                usage: 'Overwrite the bucket CORS configuration',
+                type: 'boolean'
+              },
+              confirm: {
+                usage: 'Display a confirmation prompt',
+                type: 'boolean'
+              }
```

Comment:
Sorry this merge request is a bit messy because it includes my commit from #138 (otherwise it was hard to improve the tests without having serverless 3 compatibility). Didn't add any new functionality here, just defined these existing cli options so they work with serverless 3.